src/index.html

```diff
@@ -7,8 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Raleway:ital,wght@0,100;0,400;0,600;1,900&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active ">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover">Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#tablets" class="nav__link">Tablets</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#gadgets" class="nav__link">
              Gadgets
            </a>
          </li>
       
          <li class="nav__item">
            <a href="#tablets" class="nav__link">
              Tablets
            </a>
          </li>
```

Reply:
<img width="982" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/12ee0880-db77-4130-b5d3-eb16145f5bb1">
Fix it in all places